index.html

```diff
@@ -22680,8 +22681,8 @@ <h3 id="examples_of_jukugoruby_distribution_with_intercharacter_space_expansion"
         </figcaption>
 
     </figure>
-    <p id="en-app6_5">[[[#fig_a6-12]]] are examples with the same jukugo-ruby as in [[[#fig_a6-10]]] and [[[#fig_a6-11]]] except their positions in the lines are at the line-head or at the line-end, where the base character and the ruby character at the line-head or at the line-end are both aligned.</p>
-    <p id="ja-app6_5"><span class="index" id="d18e796">[[[#fig_a6-12]]]の例</span>は，[[[#fig_a6-10]]]及び[[[#fig_a6-11]]]の例を行頭又は行末に配置し，親文字とルビ文字の先頭又は末尾をそろえる配置方法にした例である．</p>
+    <p id="en-app6_5" its-locale-filter-list="en" lang="en">[[[#fig_a6-12]]] are examples with the same jukugo-ruby as in [[[#fig_a6-10]]] and [[[#fig_a6-11]]] except their positions in the lines are at the line-head or at the line-end, where the base character and the ruby character at the line-head or at the line-end are both aligned.</p>
+    <p id="ja-app6_5" its-locale-filter-list="ja" lang="ja"><span class="index" id="d18e796">[[[#fig_a6-12]]]の例</span>は，[[[#fig_a6-10]]]及び[[[#fig_a6-11]]]の例を行頭又は行末に配置し，親文字とルビ文字の先頭又は末尾をそろえる配置方法にした例である．</p>
```

Comment:
for similar part (https://github.com/w3c/jlreq/blob/gh-pages/old/index.html#L15803), it seems custom id was removed when merging English and Japanese, we may be better to remove these - en-app6_5 and ja-app6_5?

Reply:
pending this to future.